core/domain/stats_domain.py

```diff
@@ -129,69 +160,94 @@ class StateStats(object):
     """
 
     def __init__(
-            self, total_answers_count, useful_feedback_count, total_hit_count,
-            first_hit_count, num_times_solution_viewed, num_completions):
+            self, total_answers_count_v1, total_answers_count_v2,
+            useful_feedback_count_v1, useful_feedback_count_v2,
+            total_hit_count_v1, total_hit_count_v2, first_hit_count_v1,
+            first_hit_count_v2, num_times_solution_viewed_v2,
+            num_completions_v1, num_completions_v2):
         """Constructs a StateStats domain object.
 
         Args:
-            total_answers_count: int. Total number of answers submitted to this
-                state.
-            useful_feedback_count: int. Total number of answers that received
-                useful feedback.
-            total_hit_count: int. Total number of times the state was entered.
-            first_hit_count: int. Number of times the state was entered for the
-                first time.
-            num_times_solution_viewed: int. Number of times the solution
+            total_answers_count_v1, total_answers_count_v2: int. Total number of
+                answers submitted to this state.
+            useful_feedback_count_v1, useful_feedback_count_v2: int. Total
+                number of answers that received useful feedback.
+            total_hit_count_v1, total_hit_count_v2: int. Total number of times
+                the state was entered.
+            first_hit_count_v1, first_hit_count_v2: int. Number of times the
+                state was entered for the first time.
+            num_times_solution_viewed_v2: int. Number of times the solution
                 button was triggered to answer a state.
-            num_completions: int. Number of times the state was completed.
+            num_completions_v1, num_completions_v2: int. Number of times the
+                state was completed.
         """
-        self.total_answers_count = total_answers_count
-        self.useful_feedback_count = useful_feedback_count
-        self.total_hit_count = total_hit_count
-        self.first_hit_count = first_hit_count
-        self.num_times_solution_viewed = num_times_solution_viewed
-        self.num_completions = num_completions
+        self.total_answers_count_v1 = total_answers_count_v1
+        self.total_answers_count_v2 = total_answers_count_v2
+        self.useful_feedback_count_v1 = useful_feedback_count_v1
+        self.useful_feedback_count_v2 = useful_feedback_count_v2
+        self.total_hit_count_v1 = total_hit_count_v1
+        self.total_hit_count_v2 = total_hit_count_v2
+        self.first_hit_count_v1 = first_hit_count_v1
+        self.first_hit_count_v2 = first_hit_count_v2
+        self.num_times_solution_viewed_v2 = num_times_solution_viewed_v2
```

Comment:
Add a one-line comment above this explaining why there's no v1 so that devs don't see it as an anomaly.

Reply:
Done.